syntax = "proto3";

package contracts.services.accounts;

import "contracts/services/accounts/account.proto";

message UpdateAccountBalanceRequest {
  float balance = 1;
  string account_id = 2;
}

message UpdateAccountBalanceResponse {
  Account account = 1;
}